lambdanotes/src/components/notecard.css

```diff
@@ -0,0 +1,32 @@
+.note-card {
+  overflow: hidden;
```

Comment:
This is one of the ways you've given us the preview of each note card, right? I like it.